docs/releases/5.2.txt

```diff
@@ -37,6 +37,18 @@ Minor features
 :mod:`django.contrib.admin`
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+* Form fields in the change form are now shown below their respective labels
+  instead of next to them.
+
+* Help text in the change form is now shown after the field label and before
+  the field input.
+
+* Validation errors in the change form are now shown after the help text and
+  before the field input.
```

Comment:
I think this should have some justification as to why we're doing this change, maybe "For accessibility purposes..." or something more specific